adoc/retail_chap_formulas.adoc

```diff
@@ -27,6 +27,8 @@ Once you have made changes to your formula, ensure you apply the highstate to pr
 
 The Bind formula is used to configure the boot image on a terminal.
 
+When you are configuring the bind formula, check that you are using the same fully qualified domain name (FQDN) on both the external and internal branch networks.
+If the FQDN does not match on both networks, the build host will not be recognized as a proxy server.
```

Comment:
I prefer (emphasis on changes):
>When you are configuring the bind formula, check that **Branch server is assigned** the same fully qualified domain name (FQDN) on both the external and internal branch networks.
If the FQDN does not match on both networks, the **Branch server** will not be recognized as a proxy server.